app/models/video.rb

```diff
@@ -2,4 +2,12 @@ class Video < ActiveRecord::Base
   belongs_to :category
   validates :title, presence: true
   validates :description, presence: true
+
+  def self.search_by_title(search_term)
+    unless search_term.blank?
+      where("lower(title) like ?", "%#{search_term.downcase}%").order("created_at DESC")
+    else
```

Comment:
`unless` / `else` is not really Ruby - preferred style. Checkout solution for more Ruby idiomatic code.